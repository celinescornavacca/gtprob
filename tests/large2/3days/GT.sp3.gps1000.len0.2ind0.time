113777.52user 230.15system 36:34:16elapsed 86%CPU (0avgtext+0avgdata 2629412maxresident)k
213inputs+0outputs (0major+39270515minor)pagefaults 0swaps
