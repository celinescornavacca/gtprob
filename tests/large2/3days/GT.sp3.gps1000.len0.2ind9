(((((((((A64,(A599,A600)),(((((A223,((A871,A872),A518)),A183),(((A249,((A680,A681),(A477,A478))),A182),(A451,(A654,A655)))),(((A398,(A826,A827)),((A593,(A736,A737)),((A547,(A758,A759)),A508))),((A251,(A278,((A698,A699),((A941,A942),A419)))),(((A250,(A958,A959)),((A690,A691),(A527,(A646,A647)))),((A184,(A488,(A899,A900))),((A535,(A824,A825)),A328)))))),(((A77,(A298,(A617,A618))),(A286,A287)),(((A423,A424),((((A703,A704),((A857,A858),A805)),A402),A102)),A71)))),((((((((((A651,A652),(A564,A565)),A264),A18),(((A293,(A828,A829)),((((A175,A176),((A919,A920),((A410,(A970,A971)),A147))),(((A151,(A449,A450)),(A353,A354)),A78)),((A188,(A917,A918)),A106))),A5)),((((((A117,(A211,(A317,A318))),(A343,A344)),A84),((A672,(A990,A991)),((A734,A735),A143))),((((A692,A693),A233),((A511,A512),A288)),(A639,A640))),((A129,(A160,A161)),(A299,((A420,A421),(A622,A623)))))),(((A93,((A670,A671),A520)),A76),(A35,(((A341,A342),(A405,A406)),(A682,A683))))),(((A131,A132),((A836,A837),(((A528,(A889,A890)),A300),(A633,A634)))),(((A542,A543),A330),(A360,A361)))),A0),(((((A31,((A426,A427),(A103,(A257,(A849,A850))))),((((((A489,A490),A447),(A964,A965)),A118),(((A38,(((A526,((A624,(A694,A695)),A532)),(((A830,A831),((A663,A664),A541)),(A865,A866))),(A620,A621))),A28),(((A561,A562),((A931,A932),(A935,A936))),A50))),(A81,A82))),((A16,(A198,((A808,A809),A434))),(A15,((A769,A770),(A177,A178))))),((A121,A122),((((((A386,(A614,(A705,(A750,A751)))),A197),(A239,A240)),(((A816,(A853,A854)),(A727,A728)),(A723,A724))),((A833,A834),(A591,A592))),(((A169,A170),(A502,A503)),(A173,A174))))),(((((A838,(A855,A856)),(A185,A186)),(A179,((A710,(A754,A755)),A657))),((A187,((A792,A793),A658)),((A281,A282),A135))),(A370,A371))))),(((((((A215,A216),A156),(A30,(A304,(A387,(A568,A569))))),((((A788,A789),(A960,A961)),((A608,A609),((A779,A780),A425))),((((A36,(A95,((A822,A823),A519))),((A781,A782),A525)),(A110,((((A462,((A696,A697),A554)),A340),((A629,A630),(A625,A626))),((A847,A848),A201)))),A11))),(A414,A415)),(A1,((((A37,(A903,A904)),(((A380,A381),(A923,A924)),((A743,A744),A430))),(((A376,(A403,A404)),((A686,A687),A653)),((((A181,(A199,A200)),(((A730,((A992,A993),A955)),((A867,A868),(A912,A913))),(A893,A894))),(A217,A218)),(A234,A235)))),((A311,A312),((((A242,((A715,((A879,A880),A742)),A648)),(((A887,A888),A787),(A598,(A810,A811)))),A27),A17))))),((A25,(((A585,A586),A580),(((A984,A985),(A506,A507)),(A67,(A87,(A119,((A439,A440),((A368,A369),((A463,(A719,A720)),A243))))))))),(A2,(((A279,(((A819,A820),A452),A323)),(A39,((A606,A607),A165))),((((A366,A367),(A601,A602)),A13),((A74,A75),((A65,A66),(A47,(A61,((A324,A325),(A765,A766)))))))))))),(((((A10,(A231,(A733,(A752,A753)))),(((A748,A749),((((A673,A674),(A895,A896)),(A412,A413)),((A365,(A873,A874)),A352))),(((A72,A73),(((A594,A595),(A504,A505)),((A497,(A713,A714)),(A539,A540)))),(((A869,A870),(A491,A492)),(A500,A501))))),((((A315,A316),(((A738,A739),(A943,A944)),((A445,A446),A303))),(A48,(A224,A225))),(((A58,A59),(((A803,A804),(A881,A882)),((A205,(A603,A604)),A56))),(((A294,A295),A157),((A252,(A308,(A901,A902))),(((A237,A238),(A711,A712)),(A305,A306))))))),((((((((A289,A290),(A817,A818)),(A21,(A937,A938))),(A9,(A57,(A265,A266)))),(A12,(A22,(((A574,A575),A236),(A297,(A394,(A843,A844))))))),(((((A589,A590),(A661,A662)),(A345,(A891,A892))),(A536,(A799,A800))),((((A988,A989),A729),A642),A619))),(((((A596,A597),A563),((A976,A977),(A861,A862))),((A244,A245),A136)),(A133,A134))),((((((A794,A795),A675),A453),(((A644,A645),((A783,A784),(A350,(A384,A385)))),A68)),(A796,A797)),((A610,A611),((A253,A254),(A260,A261)))))),((((((A23,(A41,A42)),(((A949,A950),A51),((((A193,(A443,A444)),(A171,A172)),(A116,(A966,A967))),(((A126,((A939,A940),A275)),(A166,((A777,A778),A272))),(A192,((A583,A584),(A515,A516))))))),((((A986,A987),((A484,A485),(A331,A332))),((A486,A487),(A158,(A550,A551)))),(A162,(A473,A474)))),((((((A700,A701),(A641,(A785,A786))),A6),(A347,A348)),((((A32,(A448,(A974,A975))),(((A44,A45),(((A637,A638),A411),(A258,A259))),(((A382,(A921,A922)),A214),((A191,(A196,(A269,A270))),(A676,A677))))),(((A962,A963),(A273,A274)),(A24,(((A152,A153),(((A883,A884),A760),A329)),(A62,(A523,A524)))))),((A863,A864),(A60,((A417,A418),(A929,A930)))))),((((A773,A774),(A968,A969)),((A313,A314),A140)),((((((A839,A840),A835),A544),A416),A97),((A336,A337),((((A915,A916),A531),((A141,A142),A120)),(((A479,A480),A232),((A441,A442),(A455,(A631,A632)))))))))),(((((((A509,A510),((A113,A114),((A139,(A437,A438)),(A296,((A798,(A947,A948)),A307))))),(A241,((A761,A762),A246))),(((A877,A878),A832),A210)),(((A668,A669),A378),A14)),(((((A52,(A53,(A111,A112))),((A209,(A291,A292)),(A189,A190))),A34),(A159,(A740,A741))),A4)),((A40,((A338,A339),A138)),(((((A806,A807),A468),(A545,(A851,A852))),(((A615,A616),A605),A383)),(((((A204,(A659,A660)),(A578,A579)),(A101,(A104,A105))),((((A558,(A841,A842)),A407),A357),((A148,(A321,(A933,A934))),(((((A635,A636),A557),A422),A399),((A702,(A790,A791)),(A555,(A909,(A982,A983)))))))),((A212,A213),A43)))))),((A26,((A247,((A372,(A612,A613)),A351)),((A665,(A688,A689)),A125))),((((A684,A685),A128),(A228,((A464,A465),(A396,A397)))),(((A301,A302),(A155,(A814,A815))),(A566,A567))))))),(((A812,A813),((A656,(A927,A928)),(A725,A726))),((((((A980,A981),(A885,A886)),(A556,(A649,A650))),((A581,A582),(A875,A876))),((A771,A772),A643)),(((((A408,A409),(A373,A374)),(((A747,(A996,A997)),(A208,(A226,A227))),(A346,(A471,A472)))),((A86,(A255,A256)),(A716,A717))),(((A46,(A349,(A481,(A493,A494)))),A20),((((A513,A514),A154),A124),(A69,A70))))))),(((((A379,(A821,(A953,A954))),((A998,A999),A522)),A8),((((A548,A549),(A587,A588)),((A271,(A377,(A907,A908))),((A248,(A951,A952)),((A956,A957),A517)))),((A731,A732),((A333,A334),((A229,A230),A123))))),(((((A309,A310),((((A109,((A401,(A482,A483)),A206)),((((A756,A757),A718),A400),A195)),A83),(A149,A150))),(A79,((A466,A467),A96))),(((((((A283,A284),(A498,A499)),(A163,A164)),(((A576,A577),(A335,(A537,A538))),(A845,A846))),((((A363,A364),(A460,A461)),(A706,A707)),A19)),((A90,A91),((A98,(A358,A359)),(A559,A560)))),((A85,(((A627,A628),((A775,A776),(A202,A203))),A130)),(A92,(A326,A327))))),((((((((A127,((A533,A534),(A395,(A801,A802)))),((A145,A146),(A432,A433))),((A529,A530),((A666,A667),((A859,A860),(A495,A496))))),(((A262,A263),A94),((A280,(A392,A393)),((A276,A277),A207)))),A7),(((A475,A476),(A220,A221)),((A521,(A721,A722)),((A469,A470),((A678,A679),(A925,A926)))))),(A319,A320)),((((((A905,A906),A285),((A219,(A454,(A978,A979))),A137)),(A458,A459)),((((A456,A457),A63),A49),(A54,A55))),((((((A945,A946),A914),((A767,A768),(A708,A709))),(A546,(A570,A571))),(((A99,A100),A80),(((((A572,A573),A222),(A107,(A431,(A972,A973)))),(A88,A89)),A29))),(((((A167,A168),A108),A33),(A322,((A897,A898),A362))),A3))))))),(((A180,(((A552,A553),A375),(A910,A911))),((((A390,(A391,(A435,A436))),(A355,A356)),(A194,(A763,A764))),(A994,A995))),((A115,((A267,A268),(A388,A389))),((A745,A746),(A144,(A428,A429)))))),(((((((B706,B707),B264),(B65,((B542,B543),B99))),(((((((B683,B684),B597),B595),B301),B199),B96),(((B725,B726),B176),B118))),(((((B49,((B288,(B419,B420)),B162)),(((((((B475,(B489,(B563,B564))),(B308,(B968,B969))),B112),B79),((B777,B778),B517)),(B103,(((B430,B431),(B754,B755)),((B690,B691),B402)))),((((B443,(B458,((B858,B859),(B590,B591)))),B28),B7),(B666,B667)))),(((B9,((((B348,B349),B335),B25),(B56,((B242,B243),(B370,B371))))),((B850,B851),((B336,B337),(B661,B662)))),(((B196,(B785,B786)),(B10,((((B395,((B748,B749),B398)),(B193,(B584,(B836,B837)))),((B696,B697),((B970,B971),B309))),((B554,B555),(B234,((B538,(B635,B636)),B358)))))),(((B210,B211),B69),((B652,B653),(((B45,B46),(B57,B58)),B43)))))),(((((B943,B944),(B656,(B659,(B791,B792)))),(B727,B728)),((((((B484,B485),(B295,(B378,B379))),(B66,(((B602,B603),B102),B71))),((B637,B638),B95)),(((((B909,B910),B521),(B362,(B515,B516))),B91),(B183,(B687,(B704,B705))))),(B126,B127))),(((((B160,(B929,B930)),((((B482,B483),B294),B241),B203)),((B359,B360),((B883,B884),(B829,B830)))),(((B490,B491),((B435,(B838,B839)),((B366,B367),(B513,B514)))),(((((B212,B213),(((B708,B709),((B621,B622),B229)),(B523,B524))),((B840,B841),B817)),(B324,(B633,B634))),((B340,B341),(B317,(B658,(B802,B803))))))),B11))),(((((((B20,((((B208,B209),(B673,B674)),(B207,(B230,(B383,B384)))),B42)),((((B759,B760),B672),(B339,(B721,B722))),((((B974,B975),B368),((B558,B559),(B113,((B302,B303),B174)))),(((((B660,((B921,B922),B880)),(B729,B730)),B197),((B399,B400),(B233,(B825,B826)))),(B376,B377))))),(((B168,(B352,(B408,B409))),((B59,(B72,(B73,((B260,B261),((B462,B463),B438))))),((B83,((B135,B136),((B310,B311),(B577,B578)))),((B316,((B972,B973),B434)),(((B866,B867),B643),(B223,(B495,B496))))))),B19)),(((B639,B640),B297),(B833,(B872,B873)))),((((B92,(B115,B116)),B16),(((B996,B997),B128),(B93,B94))),((((B810,B811),(B97,(B299,B300))),B37),(((((((B163,B164),B98),B84),((B394,((B846,B847),B518)),(B764,B765))),B30),((((B675,B676),(B527,B528)),B289),((B424,B425),((B305,(B448,B449)),(B124,B125))))),(B132,((B917,B918),B215)))))),(((B891,B892),(((B931,B932),(B510,B511)),B88)),(((((B171,((B933,B934),(B537,(B573,B574)))),(B580,B581)),(B354,B355)),((B276,(B444,B445)),B87)),((((((((B529,B530),((B771,B772),(B364,B365))),B129),(B105,B106)),B90),B54),(((B629,B630),B607),B220)),((B225,(B923,B924)),(B988,B989)))))),(((((((((B668,B669),(B919,B920)),(B198,(B860,B861))),((B156,B157),(((B412,B413),B175),(B109,B110)))),((B436,B437),(B369,(B733,B734)))),(((((B626,B627),((B895,B896),B756)),(B327,(B821,B822))),(((B426,B427),B165),B22)),((B17,((((B616,B617),B497),((B844,B845),B177)),((B145,(B502,B503)),B138))),(((B553,(B735,(B740,B741))),(B410,(B648,(B990,B991)))),(((B651,(B773,B774)),(B692,B693)),B104))))),(((B304,(B623,(B685,B686))),(((B606,(B779,B780)),(B380,((B464,B465),B407))),((((B219,(B945,(B952,B953))),(B280,B281)),(B263,(B333,B334))),((B130,B131),B70)))),((((B184,(B907,B908)),B18),(B134,(B292,(B350,(B446,B447))))),B12))),((((((B298,(B567,B568)),B53),B48),(B984,B985)),B0),((((B956,B957),(B585,B586)),(((B582,B583),(B274,B275)),B52)),(((((B107,B108),((B852,B853),B119)),B41),((B61,B62),((B284,B285),B222))),(((((B905,B906),B820),((B248,(B282,(B718,(B783,B784)))),((B500,B501),B401))),(B121,B122)),((B80,((B789,B790),B269)),(B293,((B387,(B864,B865)),(B416,(B818,B819)))))))))),(((B353,(B592,B593)),(B267,B268)),((B670,B671),((B494,(B966,B967)),(B81,B82)))))))),(((((((((((B24,(((B235,B236),((B624,B625),B548)),(B382,(B775,B776)))),B21),(((B588,B589),(B249,B250)),(B50,(B75,B76)))),((((B381,(B470,B471)),(((B712,B713),B560),(B737,(B766,B767)))),B39),((B680,(B913,B914)),B2))),(((B166,((B899,B900),B689)),(((B539,B540),B114),(((B169,B170),(B998,B999)),(((B468,B469),((B757,B758),(B525,B526))),((B870,B871),B338))))),(((((B36,((B417,B418),B226)),((((((B155,((B649,B650),(B641,B642))),(B89,((B646,B647),(B356,B357)))),((B414,B415),B405)),((B191,B192),(B986,B987))),((B187,B188),(B123,(B604,B605)))),(((B925,B926),(((B450,B451),B214),(B100,B101))),(B141,B142)))),((B55,((B575,B576),((B731,B732),(B644,B645)))),((((B571,B572),B351),B186),(B158,(B631,B632))))),((B63,(((B954,B955),B628),B587)),(((B736,(B897,B898)),B31),(B876,B877)))),((((B322,B323),B33),(B533,(B800,B801))),((((B982,B983),B534),B159),B64))))),((B3,(((B442,(B823,B824)),(B189,B190)),((((B889,(B980,B981)),(B452,B453)),((B509,(B881,B882)),B332)),((B885,B886),((B345,B346),B228))))),((B200,B201),((B421,B422),((B396,B397),((B374,B375),((B480,B481),B439))))))),(((((B389,(B392,B393)),B111),((B185,(((B271,B272),B262),((B958,B959),(B492,B493)))),(B68,(B239,(B403,B404))))),(B265,B266)),((B38,((((B202,((B831,B832),B344)),B146),(((B273,((B290,B291),B277)),(B911,B912)),((((B455,B456),B440),B423),(((B556,B557),(B750,B751)),B406)))),((B194,(((B561,B562),B347),B240)),(B178,B179)))),(((B946,B947),B682),(((((B679,(B738,B739)),((B941,B942),(B258,(B794,B795)))),(((B457,((B935,B936),B663)),(((B478,B479),(B552,((B848,B849),B596))),B361)),((B768,(B808,B809)),B551))),B47),((((B246,B247),((B600,B601),B180)),((B148,B149),(B328,B329))),((B716,B717),B227))))))),(((((B195,((B688,(B812,B813)),B512)),B44),(((B719,B720),((B474,(B950,B951)),B254)),((B252,B253),(B143,B144)))),((((B700,B701),B541),(B167,(B992,B993))),(B29,(B804,B805)))),(((((B698,B699),(B466,B467)),(B237,(B598,B599))),B26),((B13,(B161,(B318,B319))),(((B133,(B181,B182)),B117),((B286,B287),(B796,B797))))))),(((((((B411,(B612,B613)),(B677,B678)),((B77,B78),(((B320,B321),(B769,B770)),(((B498,B499),(B546,B547)),((B278,B279),((B664,B665),(B806,B807))))))),(((B432,B433),((B519,B520),B454)),((B798,B799),B330))),((B363,(((B746,B747),B681),B508)),((((B878,B879),(B506,B507)),(B939,B940)),((B742,B743),((B35,((B172,(B618,(B927,B928))),((B937,B938),(B486,(B610,B611))))),((B60,(((B781,B782),B615),B283)),B32)))))),((((B856,B857),B429),B150),(B34,(B549,B550)))),(B8,(((B723,B724),B139),((B173,(B842,B843)),(B151,B152)))))),(((((B221,((((B960,B961),B814),B325),B238)),(B787,B788)),B14),((((B887,B888),(((B504,B505),(B326,(B565,B566))),(B834,B835))),B1),((B15,(((B460,B461),(B307,(((B815,B816),(B901,B902)),B522))),((B217,B218),((B710,B711),((B255,B256),B137))))),(B153,B154)))),((((B251,(((B472,B473),((B862,B863),B763)),B388)),((B385,B386),(B231,B232))),((B4,((B51,(B205,B206)),B23)),((((B459,(B752,B753)),B204),(B224,(B441,(B976,B977)))),(B535,B536)))),(((((((B714,B715),(B962,B963)),(((B331,(B579,(B608,B609))),(B257,(((B903,B904),B890),((B762,(B854,B855)),B761)))),(B614,(B964,B965)))),B40),((B868,B869),((B314,B315),((B545,((B744,B745),B657)),B296)))),((((B312,B313),B270),(((B874,B875),B390),(B978,B979))),(B74,(B244,B245)))),((((((B702,B703),(B216,(B476,B477))),B67),(((B531,B532),B259),(B147,(B544,(B594,(B654,B655)))))),(((B342,B343),(B569,B570)),((B27,(B372,B373)),(B6,(((B994,B995),B428),B140))))),((B306,(((B915,B916),B793),B391)),((B694,B695),B5))))))),((B85,B86),(((B827,B828),B120),(((B619,B620),(B487,B488)),((B948,B949),(B893,B894))))))),(((((C629,C630),((C169,(C557,C558)),((C775,C776),C15))),((((C427,(C803,(C812,C813))),((((C693,C694),((C801,C802),C165)),((((((((C633,C634),C300),((C870,C871),(C868,C869))),(((C516,C517),C400),(C460,C461))),C122),(C167,(C593,(C790,C791)))),((C130,C131),((C160,C161),(((((C996,C997),C707),C571),C430),((C799,C800),C254))))),(((C124,((C624,C625),(C238,(C249,(C810,C811))))),C99),(C97,(C491,C492))))),C14)),(((C838,C839),((C548,C549),((C956,C957),C411))),(C667,C668))),((((C291,(C345,(C654,C655))),(C118,(C450,C451))),((((C435,(C960,C961)),((C725,C726),((C860,C861),C828))),C68),C33)),(((C34,C35),((C100,(C119,C120)),(C740,C741))),(C356,C357))))),((((C854,C855),(C145,(C552,C553))),(((C436,(C833,C834)),(C339,C340)),((C663,(C773,C774)),C207))),(((((C362,C363),(((((C421,C422),C91),C82),((((C215,(C395,C396)),C132),C128),(((C533,C534),C314),C239))),C7)),((((((C138,(C680,C681)),((C509,C510),C226)),((C74,C75),(C286,(C298,(C536,C537))))),C2),(((C115,C116),((C842,C843),(((C493,C494),C397),((((C698,C699),C617),C538),C93)))),(((C40,((C650,C651),((C768,C769),C735))),(((C702,(C759,C760)),C213),((C503,(C786,C787)),(C415,(C738,C739))))),(((C158,C159),C48),(((((C297,((C407,(C660,C661)),(C434,(C616,(C675,(C901,C902)))))),C185),C180),C61),((C186,(C899,C900)),C41)))))),(((C103,((C631,C632),(C311,C312))),((C243,C244),C197)),((C211,((C866,C867),C370)),(((C154,(C988,C989)),(C682,C683)),((C343,(C568,C569)),C228)))))),((((C83,C84),(C201,C202)),((((C24,((C905,C906),C290)),((C596,C597),C410)),C16),((((C622,C623),(C20,((C80,(((C612,C613),(C535,(C751,(C896,C897)))),C88)),((C764,C765),C184)))),((C79,((C504,(C554,C555)),(C347,(C732,C733)))),((C398,C399),(C984,C985)))),((((C455,((C771,C772),C692)),(C210,(C614,C615))),(((C382,(C388,(C676,C677))),C381),(C994,C995))),((((C171,(C349,(C425,C426))),(C602,C603)),((C36,((C236,(C927,C928)),((C932,C933),C662))),((C936,C937),C276))),(C218,C219)))))),(((C39,((C672,C673),(C585,(C678,C679)))),((C903,C904),(C367,(C478,(C665,C666))))),((((C878,C879),C137),(C125,(C187,C188))),(((C181,(C246,(C840,C841))),((((C703,C704),(C829,C830)),(C565,C566)),C114)),(C11,(C45,(C346,(C560,(C610,C611)))))))))),((((((((C127,(C360,(C844,C845))),C37),C9),((((C978,C979),((C206,((C348,(C794,C795)),C304)),(((C442,C443),C328),C172))),(C229,C230)),(C162,C163))),((((C216,C217),C157),((C437,C438),((C686,C687),C562))),(((((C723,C724),C570),((C529,(C826,C827)),C475)),((C729,C730),C325)),(C133,(C155,C156))))),(((((((C72,C73),(C917,C918)),(C182,(C250,(((C743,C744),C387),C332)))),(C23,(C705,C706))),((C848,C849),((C446,C447),((C852,C853),(((C986,C987),C608),C401))))),(C221,(C305,(C454,(C656,C657))))),((((C25,(((C525,C526),C126),C30)),((C316,(C417,C418)),(C919,C920))),(C51,(((C544,C545),(C284,C285)),(C423,C424)))),(((C929,C930),(C214,(C701,(C822,C823)))),(((C484,(C976,C977)),(C318,C319)),(C121,(C313,((C520,C521),C441)))))))),((C44,((((C882,C883),(C749,C750)),C212),(C954,C955))),((C177,C178),((C90,((((C293,C294),C194),(C710,C711)),C183)),(C648,C649))))),(((((C652,C653),C10),((C53,(C241,C242)),(((((((C752,C753),(C270,C271)),(C500,C501)),((((C251,(C456,(C690,C691))),C240),C117),(C148,C149))),(C268,((C333,C334),(C886,C887)))),((((C272,(C850,C851)),(C792,C793)),C76),((C716,C717),(C365,(C518,(C522,C523)))))),((C391,C392),(((C884,C885),(C373,(C408,C409))),(C514,(C777,C778))))))),(C546,C547)),((((((C452,C453),(C299,(C444,C445))),(C326,C327)),C108),((C471,C472),((C485,C486),C457))),((((C550,C551),((C393,C394),(C354,C355))),(C531,C532)),(((C539,C540),C265),(C38,(C42,C43)))))))))),((((((((C502,(C831,C832)),(C591,C592)),((C69,C70),(C17,((C203,(C541,(C639,C640))),((C714,C715),(C306,((C931,(C970,C971)),C638))))))),((((C358,C359),(C151,C152)),(C179,(C890,C891))),(C146,C147))),((C52,((C646,C647),C95)),((C5,((C527,C528),((C915,C916),(C317,(C600,C601))))),C1))),((((C18,C19),(((C281,(C368,(C762,C763))),((C934,C935),((C972,C973),(C862,C863)))),((C864,C865),(((C176,(C473,C474)),(C269,(C479,C480))),((C938,C939),((C736,C737),((C559,(C806,C807)),(C255,((C335,C336),(C383,C384)))))))))),((((((((C344,((C797,C798),C412)),(C112,C113)),((((C258,C259),((C482,C483),(((C150,(C816,C817)),C101),(C198,(C295,C296))))),(C64,((C755,C756),(C404,(C530,(C618,C619)))))),(((C588,C589),C481),(((C96,(C273,C274)),((C688,C689),((C950,C951),(C385,C386)))),(C98,(((C858,C859),(C669,C670)),(C247,(C277,(C580,((C734,(C892,C893)),C664)))))))))),C4),((C448,C449),(((C766,C767),C754),C742))),C0),(((C31,C32),((C29,(C166,(C590,(C990,C991)))),(C231,C232))),((C63,(C324,(C371,((C620,C621),(C641,C642))))),((((((((C796,(C974,C975)),(C574,(C876,C877))),C89),C62),(C140,(C583,C584))),(C925,C926)),(((C374,C375),(C874,C875)),((((C898,(C911,C912)),C731),C262),((C329,C330),(((C846,C847),C110),((((C856,C857),(((C942,C943),C722),C695)),((C106,(C237,((C489,C490),(C331,(C643,(C940,C941)))))),C102)),((C808,C809),(C301,(C505,C506))))))))),(C26,((((C220,((C595,(C708,C709)),C416)),((C458,C459),(C964,C965))),(C57,((C604,C605),C280))),(((C287,(C542,C543)),(C907,C908)),(((C302,C303),(C288,C289)),((((C992,C993),((C944,C945),C785)),C292),((C468,(C609,(C684,C685))),C364)))))))))),((C86,C87),(C199,C200)))),((((C921,C922),(C341,C342)),(((C282,C283),C204),(((C582,(C923,C924)),(((C894,C895),C835),C519)),(C757,C758)))),(((((C487,C488),C440),((C556,(C909,C910)),C524)),((((C962,C963),C170),(((C626,(C658,C659)),C465),C193)),C22)),(C85,(C913,C914)))))),(C8,(((C195,C196),((C56,(C781,C782)),(C81,(C109,((C245,((C718,C719),C439)),C139))))),((((C104,C105),(C208,(C275,(((C779,C780),C770),C378)))),((C189,C190),(((C470,((C606,C607),C511)),C192),(C390,(C627,C628))))),C50)))),(((((((C577,(C836,C837)),C94),((((C515,(C696,C697)),C164),(C952,C953)),((((C337,C338),(C278,C279)),(((C761,(C824,C825)),C637),C168)),(((C260,C261),(C814,C815)),((C65,C66),(C888,C889)))))),((C12,((C28,(((C107,((C496,C497),((C982,C983),C433))),((((C512,C513),C372),C227),C78)),(((C880,C881),C674),(C419,C420)))),(C92,((C134,C135),(C980,C981))))),C3)),((((C379,C380),(C58,C59)),(((((C71,(C77,(C635,C636))),(((C413,C414),C366),C223)),((((C998,C999),((C958,C959),(C948,C949))),(C191,((C788,C789),(C352,C353)))),(C54,((C476,C477),(C402,C403))))),C6),(C27,(C143,C144)))),(((C405,C406),((C67,(C320,C321)),(C49,(C783,C784)))),((C598,C599),((C141,(C745,C746)),(C123,((C575,C576),(C376,C377)))))))),(((((C431,C432),((C233,C234),(C266,C267))),(C256,C257)),(C21,((((C60,(((((C671,(C966,C967)),C248),(C309,C310)),((C175,(C572,C573)),C129)),(C142,(((C466,C467),C323),C153)))),(C804,C805)),(C55,(C586,C587))),((((C507,C508),C469),C361),(C712,C713))))),((C307,C308),(((C136,((C720,C721),((C747,C748),C462))),((C350,C351),C222)),((((C205,(C872,C873)),C47),(C369,(C495,(C578,C579)))),(C322,(C946,C947))))))),((((C111,(C252,(C563,C564))),(((((C644,C645),C235),(C727,C728)),(C253,((C818,C819),C389))),C13)),(((C463,C464),C315),C46)),(((((C561,(C567,(C581,(C594,(C820,C821))))),C264),((C968,C969),C700)),(C209,(C224,C225))),((C498,C499),((C173,C174),(C263,(C428,C429)))))))))));
