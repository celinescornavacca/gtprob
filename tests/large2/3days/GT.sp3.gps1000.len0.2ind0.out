-19254.4338359
